packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -16,6 +18,10 @@ import { obfuscateNumber } from '../sms/base'
 
 const ATTESTATION_ERROR = 'Valid attestation could not be provided'
 const NO_INCOMPLETE_ATTESTATION_FOUND_ERROR = 'No incomplete attestation found'
+export const INVALID_SIGNATURE_ERROR = 'Signature is invalid'
+
+const odisPubKey = OdisUtils.Query.ODIS_MAINNET_CONTEXT.odisPubKey
```

Comment:
(apologies for commenting this in two places, old thread was outdated)
hey one thing on this @isabellewei, we should still make the odisPubKey configurable by network (i.e. perhaps via an optional env var that specifies the network and then use that to pick the proper ODIS context), otherwise I guess the logic will fail on alfajores/baklava?

Reply:
I ended up adding an env variable and using it with `getServiceContext`. Let me know if that looks good!